src/Models/DomainRouter.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Cyberfusion\ClusterApi\Models;
+
+use Cyberfusion\ClusterApi\Support\Arr;
+use Cyberfusion\ClusterApi\Contracts\Model;
+
+class DomainRouter extends ClusterModel implements Model
+{
+    private string $domain;
+    private bool $forceSsl = true;
+    private ?int $virtualHostId = null;
+    private ?int $urlRedirectId = null;
+    private ?int $nodeId = null;
+    private ?int $id = null;
+    private ?int $clusterId = null;
```

Comment:
When I look at the documentation, I'm seeing a GET and PUT for this. The PUT requires the `id` and `cluster_id`. A GET always returns those values. So can we remove the nullable property from these properties?

```suggestion
    private int $id;
    private int $clusterId;
```

Reply:
Done in 33fa6ea2939fe757da73a24d061282bf0a67e4f1